packages/column/package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "@udecode/plate-layout",
+  "name": "@udecode/plate-column",
```

Comment:
We can keep `@udecode/plate-layout` package name so other plugins may fit in the future.